packages/design-system-v2/src/Radio/Radio.styles.tsx

```diff
@@ -0,0 +1,111 @@
+import styled, { css } from "styled-components";
+
+const Variables = css`
+  --radio-color-border: var(--ads-v2-color-border);
+  --radio-color-background: transparent;
+  --radio-color-border-checked: var(--ads-v2-color-bg-brand-secondary);
+  --radio-color-check-mark: var(--ads-v2-color-bg-brand-secondary);
+  --radio-color-label: var(--ads-v2-color-fg);
+`;
+
+const BasicStyles = css`
+  input[type="radio"] {
+    position: absolute;
+    opacity: 0;
+  }
+
+  label {
+    cursor: pointer;
+    position: relative;
+    padding-left: var(--ads-v2-spaces-7);
+    font-family: var(--ads-v2-font-family);
+    color: var(--radio-color-label);
+  }
+
+  label::before {
+    content: "";
+    position: absolute;
+    width: var(--ads-v2-spaces-5);
+    height: var(--ads-v2-spaces-5);
+    background-color: var(--radio-color-background);
+    border: 1px solid var(--radio-color-border);
+    border-radius: 50%;
+    top: 50%;
+    left: 0;
+    transform: translateY(-50%);
+    transition: border-color 400ms ease;
+  }
+
+  label::after {
+    content: "";
+    position: absolute;
+    width: var(--ads-v2-spaces-5);
+    height: var(--ads-v2-spaces-5);
+    background-color: var(--radio-color-check-mark);
+    border: 1px solid var(--radio-color-check-mark);
+    border-radius: 50%;
+    top: 50%;
+    left: 0;
+    transform: translateY(-50%) scale(0);
+    transition: transform 200ms ease;
+  }
+
+  input[type="radio"]:checked + label::before {
+    border-color: var(--radio-color-border-checked);
+  }
+
+  input[type="radio"]:checked + label::after {
+    transform: translateY(-50%) scale(0.55);
+  }
+`;
+
+export const RadioComponent = styled.label<{
```

Comment:
Why are we continuing to use the suffix Component in our variable names?

Reply:
This cannot be called as Radio since the Radio.tsx file is already have a component in name Radio.

Reply:
What's wrong with StyledRadioLabel?